code/game/objects/items/contraband_vr.dm

```diff
@@ -36,12 +36,12 @@
 						/obj/item/device/pda/mime,
 						/obj/item/device/pda/syndicate,
 						/obj/item/mecha_parts/chassis/phazon,
-						/obj/item/mecha_parts/part/phazon_head,
-						/obj/item/mecha_parts/part/phazon_left_arm,
-						/obj/item/mecha_parts/part/phazon_left_leg,
-						/obj/item/mecha_parts/part/phazon_right_arm,
-						/obj/item/mecha_parts/part/phazon_right_leg,
-						/obj/item/mecha_parts/part/phazon_torso,
```

Comment:
hm, although spades. the actual chance to get all the phazon pieces, even with something as much as 40 crates is so low that getting the whole set is like winning the lottery, exponential chance that is. I could math and put it all out for you, but the numbers wouldn't be very encouraging.